libpod/rlimit_uint64.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux || darwin
-// +build linux darwin
+//go:build !remote && linux
+// +build !remote,linux
```

Comment:
I'll assume the removal of `darwin` is intentional, and everything will work fine, but will mark this as perhaps needing extra attention just in case.

Reply:
yes, I mean it makes no difference either way.
libpod does not work/compile on darwin so I just dropped it.